frontera/settings/default_settings.py

```diff
@@ -50,6 +51,7 @@
 }
 SQLALCHEMYBACKEND_REVISIT_INTERVAL = timedelta(days=1)
 STATE_CACHE_SIZE = 1000000
+STATE_CACHE_SIZE_LIMIT = 0
```

Comment:
Do you think we should use a different value other than 0? For hbase it is - https://github.com/Preetwinder/frontera/blob/master/frontera/settings/default_settings.py#L21